index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/responsive.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div id="container">
+      <header>
+        <nav>
+          <p id="logo">Lisandro Seia</p>
+
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <section id="present">
+          <div id="titles">
+            <h1>
+              I’m Lisandro <br />
+              Glad to see you!
+            </h1>
+            <p id="present-text">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div id="contact">
+            <p id="contact-p">LET’S CONNECT</p>
+            <div id="icons">
+              <a target="blank" href="https://twitter.com/LisandroSeia"
+                ><img alt="twitter" src="/assets/imgs/twitter.png"
+              /></a>
```

Comment:
- [ ] these icons section should be implemented using a `ul` design. You are using a div here. Please also make sure that all the `a` links are wrapped inside `li` tags. 
Something similar to this would be a lot better.
```suggestion
           <ul id="icons">
              <li>
                <a target="blank" href="https://twitter.com/LisandroSeia">
                  <img alt="twitter" src="/assets/imgs/twitter.png"/>
                </a>
              </li>
```